internal/pkg/template/templates/workloads/services/static-site/manifest.yml

```diff
@@ -0,0 +1,43 @@
+# The manifest for the "{{.Name}}" service.
+# Read the full specification for the "{{.Type}}" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/static-site/
+
+# Your service name will be used in naming your resources like S3 buckets, etc.
+name: {{.Name}}
+type: {{.Type}}
+
+
+{{- if not .FileUploads}}
+# files:
+#   - source: ./static
+#     destination: static
+{{- else}}
+files:
+{{- range $file := .FileUploads}}
```

Comment:
Sorry I am dumb, I am not able to understand why we are looping over FileUploads and setting source, destination and so on same in every loop?

Reply:
Don't say that ! it's not at all dumb

My understanding is that `.FileUploads` is more like "directories". For example, if we wanted to upload 2 different directories this would translate to:
1. First, upload `source: 'src/static/images'` to the S3 bucket under `destination: 'assets/'`
2. Then, upload `source: 'src/static/js'` to the S3 bucket under `destination: 'assets/'`

Hope that helps understands what the field represents!